rust/Dockerfile.test

```diff
@@ -7,8 +7,6 @@ RUN rustup default nightly-2020-05-18
 # Build cargo cache
 RUN cargo build
 
-RUN rustup default nightly-2020-05-18
```

Comment:
This change might break local tests? Due to rocket requiring nightly

Reply:
Hey Griffin, it turns out that rustup was installed twice, check line 6: 

https://github.com/security-union/armore-server/pull/26/files#diff-fcd9de863d96a06813721bb15578242126badfc69fc4ceb60c7ac379b34481ccR6